wifiphisher/pywifiphisher.py

```diff
@@ -1024,7 +1050,10 @@ def run():
     clients_APs = []
     APs = []
     args.accesspoint = ap_mac
-    args.channel = channel
+
+    if not (args.apname and args.channel):
+        args.channel = channel
```

Comment:
Yes this is good because currently if only one of the arguments is supplied it falls back to normal operation.